media/jsonp/common/src/main/java/io/helidon/media/jsonp/common/JsonpSupport.java

```diff
@@ -61,6 +62,19 @@ public JsonpBodyWriter newWriter() {
         return new JsonpBodyWriter(jsonWriterFactory);
     }
 
+    /**
+     * Create a new JSON-P stream writer.
+     * <p>
+     * This stream writer supports {@link java.util.concurrent.Flow.Publisher publishers}
+     * of {@link javax.json.JsonStructure} (such as {@link javax.json.JsonObject})
+     * , writing them as an array of JSONs.
+     *
+     * @return JSON processing stream writer.
+     */
+    public JsonpBodyStreamWriter newStreamWriter() {
```

Comment:
The newXXX static methods have been removed. Let's remove it.

Reply:
Actually it is not removed in this version. I didn't commit these changes yet.